protected/humhub/libs/ClassMapCacheModule.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace humhub\libs;
+
+class ClassMapCacheModule
+{
+    //  public properties
+    public ?int   $id = null;
+    public string $class_name;
+    public string $moduleId;
+
+    public function __construct(
```

Comment:
All one line?
```php
public function __construct($values,  string $class_name = '', string $moduleId = '')
```

Reply:
Hi @ArchBlood, thank you for taking the time to have a look at this and please accept my apologies for not having followed the project's code style guidelines. Of course, happy to apply those if the PR in general is welcomed. Do you use only CSFixer? Or is there also a repo for shared `.idea`-settings?